tests/distributed/test_local_flow_use_remote_executor/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.3"
+services:
+  external-executor:
+    image: jinaai/jina:test-pip
+    environment:
+      JINA_LOG_LEVEL: DEBUG
+    container_name: test_external_executor
+    ports:
+      - "8001:8000"
+      - "45678:45678"
+    expose:
+      - 10000-60000
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    entrypoint: "jina executor --uses jinahub+docker://CLIPImageEncoder --port-in 45678 --socket-in ROUTER_BIND --dynamic-routing-out"
```

Comment:
jinahub://CLIPImageEncoder cannot be a dependency of the `test`. There is no need to test this specific executor. U can create one if u want for the test but is overkill